<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>450</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(194, 237, 206);</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">#order_id_label  , #product_name_label , #order_status_label , #delivery_address_label {
background-color:white;
border-radius:10px;
padding-left:10px;
}
 #update_status_lineedit{
background-color:white;
padding-left:10px;
}</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>626</width>
        <height>586</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>550</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <widget class="QLabel" name="image_label">
          <property name="geometry">
           <rect>
            <x>120</x>
            <y>30</y>
            <width>361</width>
            <height>200</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>200</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel#image_label{
background-color:white
}</string>
          </property>
          <property name="text">
           <string>Image Here</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>260</y>
            <width>91</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Order Id</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
         <widget class="QLabel" name="order_id_label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>260</y>
            <width>371</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Order Id</string>
          </property>
         </widget>
         <widget class="QLabel" name="product_name_label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>300</y>
            <width>371</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Product Name</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>300</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Product Name</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>340</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Order Status</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
         <widget class="QLabel" name="order_status_label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>340</y>
            <width>371</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Order status</string>
          </property>
         </widget>
         <widget class="QLineEdit" name="update_status_lineedit">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>430</y>
            <width>401</width>
            <height>31</height>
           </rect>
          </property>
          <property name="placeholderText">
           <string>Type New Order status</string>
          </property>
         </widget>
         <widget class="QPushButton" name="update_order_status_btn">
          <property name="geometry">
           <rect>
            <x>410</x>
            <y>430</y>
            <width>121</width>
            <height>28</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(102, 187, 51);</string>
          </property>
          <property name="text">
           <string>Update Status</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>480</y>
            <width>111</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">padding-left:10px;
background-color:rgb(255, 255, 0)</string>
          </property>
          <property name="text">
           <string>Order Tracker</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>380</y>
            <width>101</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Delivery Address</string>
          </property>
          <property name="margin">
           <number>5</number>
          </property>
         </widget>
         <widget class="QLabel" name="delivery_address_label">
          <property name="geometry">
           <rect>
            <x>110</x>
            <y>380</y>
            <width>371</width>
            <height>21</height>
           </rect>
          </property>
          <property name="text">
           <string>Delivery Address</string>
          </property>
         </widget>
         <widget class="QWidget" name="order_tracker_widget" native="true">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>541</y>
            <width>491</width>
            <height>80</height>
           </rect>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
